slackbridge/bots.py

```diff
@@ -149,6 +151,30 @@ def nickserv_auth(self):
                 self.target_group_nick,
                 self.nickserv_password,
             ))
+    def privmsg(self, user, channel, message):
+        """
+        Set to handle if channel is own name (private chat)
```

Comment:
I'd probably just put this on a single line like this (`"""Set to handle if channel is own name (private chat)"""`) unless you're going to add more of a description here later? I think this description is also a little confusing as-is, I'd probably say something like `Handler if a private message is received by an IRC user bot` or something like that. Thoughts?

Reply:
agreed, will rewrite to make clearer